src/sentry/static/sentry/app/views/eventsV2/utils.tsx

```diff
@@ -222,9 +222,8 @@ export function getAggregateAlias(field: string): string {
   }
   return field
     .replace(AGGREGATE_PATTERN, '$1_$2')
-    .replace('.', '_')
-    .replace(/_+$/, '')
-    .toLowerCase();
+    .replace(/\./g, '_')
+    .replace(/_+$/, '');
```

Comment:
@markstory according to docs,  if `pattern` is a string, only the first occurrence will be replaced. I made this a regex.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace

Reply:
Oops. This was for line 225